### ISSUE TITLE
TypeError: message is not a function starting with 0.9.0

### ISSUE THREAD

**Issue by ocsurfnut** [synapse-wireless-labs/jasmine-marbles#74]
When upgrading to `0.9.0`, we're now seeing this error across many of our tests:

```
TypeError: message is not a function

357 |
358 |       // Expect final state
> 359 |       expect(store.pipe(select(fromStore.selectAll))).toBeObservable(cold('a', { a: reports }));
|                                                       ^
360 |
361 |       flush();
362 |     }));

       at getMessage (node_modules/expect/build/index.js:174:15)
       at processResult (node_modules/expect/build/index.js:290:25)
       at Object.throwingMatcher [as toBeObservable] (node_modules/expect/build/index.js:359:16)
       at src/app/features/reports/reports.module.spec.ts:359:55
       at fakeAsyncFn (node_modules/zone.js/bundles/zone-testing-bundle.umd.js:5510:34)
       at ZoneDelegate.Object.<anonymous>.ZoneDelegate.invoke (node_modules/zone.js/bundles/zone-testing-bundle.umd.js:407:30)
       at ProxyZoneSpec.Object.<anonymous>.ProxyZoneSpec.onInvoke (node_modules/zone.js/bundles/zone-testing-bundle.umd.js:3765:43)
       at ZoneDelegate.Object.<anonymous>.ZoneDelegate.invoke (node_modules/zone.js/bundles/zone-testing-bundle.umd.js:406:56)
       at Zone.Object.<anonymous>.Zone.run (node_modules/zone.js/bundles/zone-testing-bundle.umd.js:167:47)
       at Object.wrappedFunc (node_modules/zone.js/bundles/zone-testing-bundle.umd.js:4250:34)
```

We run `jest` `27.0.4` along with `jest-preset-angular` `9.0.4`.  We've reverted back to `0.8.1` for now.

**Comment by djmcgrath101** [comment]
I am also getting this error but in my case I am still using version 0.8.1 with `jest` `26.2.2` and `jest-preset-angular` `8.3.2`.  Any help appreciated.

**Comment by brandonroberts** [comment]
This should be resolved by https://github.com/synapse-wireless-labs/jasmine-marbles/pull/78 and released in `0.10.0-beta.0`
